apis/python/src/tiledb/vector_search/object_api/embeddings_ingestion.py

```diff
@@ -121,14 +122,9 @@ def setup(
         # UDFs
         # --------------------------------------------------------------------
         def compute_embeddings_udf(
-            object_reader_source_code: str,
-            object_reader_class_name: str,
-            object_reader_kwargs: Dict,
-            object_embedding_source_code: str,
-            object_embedding_class_name: str,
-            object_embedding_kwargs: Dict,
+            object_index_uri: str,
             partition_dicts: List[Dict],
-            embeddings_uri: str,
+            use_updates_array: bool,
```

Comment:
(Curious) Are these functions not able to use `use_updates_array` from the top level scope? Is it b/c they're used in a DAG, i.e. they need all their data provided to them?